common/services/prismic/exhibitions.js

```diff
@@ -30,10 +31,27 @@ import {parseInstallationDoc} from './installations';
 import {london} from '../../utils/format-date';
 import {getPeriodPredicates} from './utils';
 import type {Period} from '../../model/periods';
+import type {Resource} from '../../model/resource';
 
 const startField = 'my.exhibitions.start';
 const endField = 'my.exhibitions.end';
 
+function parseResourceTypeList(fragment: PrismicFragment[], labelKey: string): Resource[] {
+  return fragment
+    .map(label => label[labelKey])
+    .filter(Boolean)
+    .filter(label => label.isBroken === false)
+    .map(label => parseResourceType(label.data));
+}
+
+function parseResourceType(fragment: PrismicFragment): Resource {
+  return {
+    title: asText(fragment.title),
+    description: fragment.description,
+    icon: fragment.icon
+  };
+}
```

Comment:
To be really bikesheddy, as these aren't used more than once, nor tested, they could just be in the parse function...